kairon/shared/log_system/base.py

```
@@ -0,0 +1,16 @@
+import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)